apps/test/unit/code-studio/pd/workshop_enrollment/enroll_formTest.js

```diff
@@ -267,18 +421,18 @@ describe('Enroll Form', () => {
       enrollForm.setState({role: 'Counselor'});
       expect(enrollForm.find('#describe_role')).to.have.length(0);
     });
+
+    it('submits when all required params are present', () => {
+      testSuccessfulSubmit(enrollForm, baseParams);
+    });
   });
 
   describe('Enroll Form', () => {
```

Comment:
can you be more specific about what kind of form this is? the outer describe block suggests these are all "enroll forms"

Reply:
Good call. All of these are testing parts of the form that are present on all types of forms ––but we aren't really testing this behavior is working on all permutations of course and subject. Just changed this block title to try to make that more clear. Open to other suggestions for this!